File: Latihan3/Latihan3_CPP/Disk.cpp
#include <string>

using namespace std;

class Disk
{
private:
	/* data */
	// class attribute
	string type;
	string capacity;
	int price;

public:
	Disk(/* args */){};
	// constructors
	Disk(string type, string capacity, int price)
	{
		this->type = type;
		this->capacity = capacity;
		this->price = price;
	}

	// getter and setter
	void setType(string type)
	{
		this->type = type;
	};

	string getType()
	{
		return this->type;
	};

	void setCapacity(string capacity)
	{
		this->capacity = capacity;
	};

	string getCapacity()
	{
		return this->capacity;
	};

	void setPrice(int price)
	{
		this->price = price;
	};

	int getPrice()
	{
		return this->price;
	}

	// destructors
	~Disk(){};
};

File: Latihan3/Latihan3_PHP/Disk.php
<?php

class  Disk
{
	// class attribute
	private $type;
	private $capacity;
	private $price;

	// constructors
	public function Disk($type = "", $capacity = "", $price = 0)
	{
		$this->type = $type;
		$this->capacity = $capacity;
		$this->price = $price;
	}

	// getter and setter
	public function setType($type)
	{
		$this->type = $type;
	}

	public function getType()
	{
		return $this->type;
	}

	public function setCapacity($capacity)
	{
		$this->capacity = $capacity;
	}

	public function getCapacity()
	{
		return $this->capacity;
	}

	public function setPrice($price)
	{
		$this->price = $price;
	}

	public function getPrice()
	{
		return $this->price;
	}

	// destructors
	function __destruct()
	{
	}
}
